crates/xwt-web-sys/src/lib.rs

```diff
@@ -347,6 +347,13 @@ impl Session {
     }
 }
 
+impl xwt_core::session::datagram::MaxSize for Session {
+    fn max_datagram_size(&self) -> Option<usize> {
+        let max_datagram_size = self.transport.datagrams().max_datagram_size();
+        Some(usize::try_from(max_datagram_size).expect("u32 should fit in a usize on WASM"))
```

Comment:
```suggestion
        Some(usize::try_from(max_datagram_size).unwrap()) // u32 should fit in a usize on WASM
```

A bit, but imp we don't want to add strings bloat to the final binaries, as they might be size-sensitive with WASM